modules/nw_multinetwork-host-device.adoc

```diff
@@ -0,0 +1,70 @@
+// Module name: nw_multinetwork-host-device.adoc
+// Module included in the following assemblies:
+// * networking/managing_multinetworking.adoc
+
+[id='multinetwork-host-device-{context}']
+= Configuring additional interfaces using host devices
+
+The host-device plugin connects an existing network device on a node directly to a pod.
+
+The code below creates a dummy device using a dummy module to back a virtual device, and assigns the dummy device `name` to `exampledevice0`.
+
+----
+$ modprobe dummy
+$ lsmod | grep dummy
+| ip link set name exampledevice0 dev dummy0
+----
+
+Connect the dummy network device to a pod:
+
+.Procedure
+
```

Comment:
I would move the "Connect the dummy network device to a pod:" line into the procedure itself